cmd/cloud.go

```diff
@@ -241,10 +244,26 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
 			return nil
 		}
 
+		shouldExitLoop := false
```

Comment:
Maybe we can directly wait on `globalCtx.Done()` instead of checking this boolean ... which also seems like a race condition

Reply:
Ah, yeah, I missed this... :sweat_smile: Not really a scary data race, but a data race nonetheless :disappointed: :+1: for directly checking the `globalCtx.Done()` in the `select` below

Reply:
@na-- I fixed the code per the comments above.  If anything more needs to change, please do tell.